modules/fts/pages/fts-troubleshooting.adoc

```diff
@@ -109,6 +109,18 @@ For If searching for the following is slow: lowFreqTerm highFreqTerm, try +lowFr
 +
 Although it is a different search, it should perform better and may be acceptable.
 
+*Q: Why are the queries getting rejected with http status code 429?*
+
+At a higher level, this shows that the cluster is under provisioned in terms of the ftsMemoryQuota. Whenever FTS service receives a query, it tries to forecast the memory requirements for performing that search request. And the query is rejected when fts service realises that the cumulative memory usage of the service goes beyond the set memory quota limit by executing the incoming query. 
```

Comment:
At a high level, the HTTP status code 429 indicates that ftsMemoryQuota for the cluster is under-provisioned.

Reply:
Replace "fts service" with "Search service"